storage-node/src/services/sync/synchronizer.ts

```diff
@@ -45,15 +44,13 @@ export async function performSync(
   selectedOperatorUrl?: string
 ): Promise<void> {
   logger.info('Started syncing...')
-  const [model, files] = await Promise.all([getStorageObligationsFromRuntime(qnApi, buckets), getDataObjectIDs()])
+  const model = await getStorageObligationsFromRuntime(qnApi, buckets)
 
   const required = model.dataObjects
 
-  const added = _.differenceWith(required, files, (required, file) => required.id === file)
-  const removed = _.differenceWith(files, required, (file, required) => file === required.id)
+  const added = required.filter((obj) => !isDataObjectIdInCache(obj.id))
 
   logger.debug(`Sync - new objects: ${added.length}`)
-  logger.debug(`Sync - obsolete objects: ${removed.length}`)
```

Comment:
I think it's good to have this log, especially if the cleanup service is disabled so that you can know about the state/status of the data objects in your node. WDYT?

For example, operator misconfigured their node with the incorrect bucket, so the log `Sync - new objects: ` probably wont provide us any useful info but `Sync - obsolete objects:`  will show that all/most  of the local data objects are obsolete?

Reply:
Good point. Will revert.